jvcl/run/JvSpin.pas

```diff
@@ -1905,6 +1930,13 @@ constructor TJvUpDown.Create(AOwner: TComponent);
   Position := 0;
 end;
 
+function TJvUpDown.GetDefaultButtonWidth: Integer;
+begin
+  result := DefBtnWidth(Screen.MonitorFromRect(
+                          TRect.Create(Left, Top,
+                                       Left + Width, Top + Height)).PixelsPerInch);
```

Comment:
`PixelsPerInch` appeared with RS 10 Seattle. Before that it's only available directly on the `TScreen` object
So the three lines should be rewritten like this:

    Result := DefBtnWidth(Screen.
      {$IFDEF RTL300_UP}
      MonitorFromRect(TRect.Create(Left, Top, Left + Width, Top + Height)).
      {$ENDIF RTL300_UP}
      PixelsPerInch;